content/en/continuous_integration/pipelines/awscodepipeline.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Set up Tracing on a AWS CodePipeline Pipeline
+kind: documentation
+aliases:
+  - /continuous_integration/setup_pipelines/codepipeline
+further_reading:
+    - link: "/continuous_integration/pipelines"
+      tag: "Documentation"
+      text: "Explore Pipeline Execution Results and Performance"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+## Compatibility
+
+- **Partial pipelines**: View partially retried executions
+
+## Configure the Datadog integration
+
+To set up the integration between [AWS CodePipeline][1] and Datadog CI Visibility, two AWS resources need to be created:
+1. [API Destination][2]: an HTTP endpoint pointing to Datadog's intake.
+2. [AWS EventBridge Rule][3]: a rule that forwards CodePipeline events to the API Destination.
+
+The following guide separates the two creation steps for clarity, but note that it's not required to create the resources
+separately. They can also be created at the same time during the EventBridge Rule creation process.
+
+More information about monitoring pipeline events can be found in the [official AWS guide][4].
```

Comment:
```suggestion
You can create these resources separately, or at the same time. For more information about monitoring pipeline events, see the [official AWS guide][4].
```

Reply:
And here perhaps we can re-add the "during the EventBridge Rule creation process" from above:

```suggestion
You can create these resources separately, or at the same time, during the EventBridge Rule creation process. For more information about monitoring pipeline events, see the [official AWS guide][4].
```

what do you think?

Reply:
LGTM